README.md

```diff
@@ -10,28 +10,30 @@ Clone FoodAdvisor
 git clone https://github.com/strapi/foodadvisor.git
 ```
 
-## 2. Start Strapi from the ./api folder
+Navigate to your *./foodadvisor* folder by running `cd foodadvisor` from your command line.
```

Comment:
```diff
- Navigate to your *./foodadvisor* folder by running `cd foodadvisor` from your command line.
+ Navigate to your `./my-projects/foodadvisor` folder by running `cd foodadvisor` from your command line.
```

I don't think that *./foodadvisor* is the right path to indicate due to the catch that we mention in **step 1** `./my-projects` path using `./`.
So the usage of `./` for `./foodadvisor` is a bit confusing.

Second point, is that you used *italic* style for a path and after the `code` style is used.
I think we should use `code` here.

Reply:
Thanks for the catch, will fix it.